### ISSUE TITLE
Numbers being parsed as date

### ISSUE THREAD

**Issue by milafrerichs** [densitydesign/raw#108]
Hi,

nice library and online tool. But one thing I ran into when I wanted to try it out is that a number got parsed as a date.
How can I change it to be a number instead?
```
City (Source),Downtown Pop,2015 City Pop, % of City
Atlanta,25376,463878,5
```
<img width="326" alt="bildschirmfoto 2017-01-17 um 18 40 54" src="https://cloud.githubusercontent.com/assets/688980/22032373/7f607db2-dce4-11e6-9fbd-19cb8c2190df.png">

Thanks.

**Comment by uf0** [comment]
Hi, we are aware of dates parsing problem, we are working on it. #106 

a quick workaround is to add a decimal to the numbers

```
City (Source),Downtown Pop,2015 City Pop, % of City
Atlanta,25376.0,463878.0,5
```

**Comment by milafrerichs** [comment]
hi @uf0 
thanks. hope you find a solution soon. for a small dataset this is feasible but not for other large datasets I have. but thanks for the hint.

**Comment by uf0** [comment]
We committed a temporary patch 068ba0cb55d7180230ddaead4348a128677f4613, now it should work also with your original dataset.

Btw we will leave the issue open until we find a more consistent solution.

**Comment by mikima** [comment]
fixed in version 1.2.0
